src/sentry/sentry_metrics/indexer/postgres/postgres_v2.py

```diff
@@ -95,17 +108,37 @@ def _bulk_create_with_retry(
     def bulk_record(
         self, use_case_id: UseCaseKey, org_strings: Mapping[int, Set[str]]
     ) -> KeyResults:
-        db_read_keys = KeyCollection(org_strings)
+        res = self._uca_bulk_record({REVERSE_METRIC_PATH_MAPPING[use_case_id]: org_strings})
```

Comment:
I may have missed some PRs here, but is `use_case_id` here referring to the configuration (release health indexer vs generic metrics indexer) or is it the use case provided together with each and every metric by the product ? 
If it is the second, why would we run one separate call to `bulk_record` per use case? The same batch of metrics can contain metrics from multiple separate use cases. 
If we run independent bulk record we are likely going to damage our throughput.

Reply:
It is referring to the configuration